src/views/preview/share-banner.scss

```diff
@@ -7,14 +7,27 @@ $navigation-height: 50px;
     width: 100%;
     overflow: hidden;
     color: $ui-orange;
+
+    /* overwrite styles for flex row and preview row */
+    .flex-row {
+        flex-direction: row;
+    }
+
+    .preview-row {
+        margin-top: 0 !important;
+        padding: .5rem;
+        align-items: center !important;
```

Comment:
We shouldn't need `!important` except as a last resort. I will play around with some ways to avoid this.